huaweicloud/services/dcs/resource_huaweicloud_dcs_instance.go

```diff
@@ -601,8 +601,9 @@ func resourceDcsInstancesCreate(ctx context.Context, d *schema.ResourceData, met
 		return diag.FromErr(err)
 	}
 
-	// create whiteList if configured.
-	if d.Get("whitelists").(*schema.Set).Len() > 0 {
+	// create whitelist when the function is enabled and has configurations
```

Comment:
```suggestion
	// create whitelist when the function is enabled and configured.
```
`is` and `has` are conflicting.